src/cljdoc/server/ingest.clj

```diff
@@ -14,10 +14,11 @@
 (defn ingest-cljdoc-edn
   "Store all the API-related information in the passed `cljdoc-edn` data"
   [storage {:keys [codox group-id artifact-id version] :as cljdoc-edn}]
-  (let [artifact (pom/artifact-info (pom/parse (:pom-str cljdoc-edn)))]
+  (let [project (util/clojars-id cljdoc-edn)
+        artifact (util/version-entity project version)]
```

Comment:
I think this change makes sense but we should also document it in the `cljdoc.cli` docs. Something like "The version you specify via the CLI will take precedence over the version specified in the POM file".

Reply:
Done. Please check if changes I've made to `ingest` command doc is ok :)